requirements.txt

```diff
@@ -4,5 +4,7 @@ sh
 # Testing
 pytest<5.0.0,>=3.3.0
 pytest-cookies
+pytest-cov
 pycodestyle
 ruamel.yaml
+sphinx_rtd_theme
```

Comment:
This one is a bit surprising. Shouldn't we install the generated package's dev dependencies before running the test that test whether we can build documentation?